mozci/console/commands/check_backfills.py

```diff
@@ -172,9 +178,8 @@ def group_key(task):
             notification = NOTIFICATION_BACKFILL_GROUP_COMPLETED.format(
                 th_symbol=th_symbol,
                 push=newest_push,
-                extra_with_twentieth_parent=f"&fromchange={parents[-1].rev}"
-                if parents
-                else "",
+                fromchange=f"&fromchange={parents[-1].rev}" if parents else "",
+                searchstr=f"&searchStr={all_backfill_tasks.pop().label.split('-e10s')[0]}",
```

Comment:
@Archaeopteryx @jmaher @ahal do you think this works for all task labels? (the goal is removing `e10s` and the chunk number from the label to filter tasks on treeherder as @Archaeopteryx suggested)

Reply:
This e.g. won't cover builds. Treeherder's [code for this](https://github.com/mozilla/treeherder/blob/bf87a56cfbe1b3ec0aad0a0fb8bf8229eddb31ea/ui/helpers/job.js#L233) also doesn't work for every task.
```
all_backfill_tasks.pop().label..replace(/(-e10s)?(-\d+)?$/, "")
```
would show all chunks in that group for that test suite.

Reply:
I think we are backfilling tests, not builds.  the two exceptions I can think of is jittest and mochitest-chrome where we do not run those in e10s, but 1proc and we chunk them:
https://treeherder.mozilla.org/jobs?repo=mozilla-central&tier=1%2C2%2C3&searchStr=jittest&revision=b1a6d489400499979f16cc982901f78ac504bc77

https://treeherder.mozilla.org/jobs?repo=mozilla-central&selectedTaskRun=FeW_ngbhS0uNBGyZmCC1lg.0&tier=1%2C2%2C3&searchStr=1proc%2Cchrome&revision=b1a6d489400499979f16cc982901f78ac504bc77

Reply:
All right, so we should exclude both e10s and 1proc, so I guess `all_backfill_tasks.pop().label.replace(/(-e10s)?(-1proc)?(-\d+)?$/, "")` should work.